.github/workflows/maven-publish.yml

```diff
@@ -2,6 +2,9 @@ name: Maven Package
 
 on:
   push:
+  pull_request:
+    branches:
```

Comment:
I  am not exactly sure but this might allow unauthorized people to run malicious PRs

Reply:
Canceling